eclipse/che
# Deprecate namespace strategies other than 'per user'

### What does this PR do?
Deprecate namespace strategies other than 'per user'

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
Fixes #19365 


### How to test this PR?
 - deploy che server image `quay.io/skabashn/che-server:che19365`
 - Setup namespace strategies other than 'per user'
 - you should see a warning message in the logs.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
